bigframes/core/groupby/__init__.py

```diff
@@ -88,9 +88,12 @@ def __getitem__(
             keys = list(key)
         else:
             keys = [key]
-        columns = [
-            col_id for col_id, label in self._col_id_labels.items() if label in keys
-        ]
+
+        for col_id, label in self._col_id_labels.items():
+            if label in keys:
+                columns = [col_id]
```

Comment:
The `col_id` should be append to the `columns` by calling `columns.append(col_id)`. Also, `columns=[]` need to be called to initialize it first.